dashboard/app/controllers/curriculum_tracking_pixel_controller.rb

```diff
@@ -14,25 +14,45 @@ def index
       # For ease of querying we attempt to parse the url into meaningful chunks.
       # EXAMPLE:
       # /csf-18/pre-express/11/
+      # es-mx/csf-1718/coursec/10/'
       split_url = curriculum_page.split('/')
       # ["", "csf-18", "pre-express", "11"]
+      # ["", "es-mx", "csf-1718", "coursec", "10"]
 
+      non_en = split_url[1][2] == "-"
```

Comment:
This feels hacky, but I wanted to avoid having to maintain a list of locales or valid "csx" values. Locales are always "xx-yy". Other ideas?

Reply:
this seems a bit too loose. Can you match it against a regex like `^[a-z][2]-[a-z][2]$`?